Libraries/acdriver/CMakeLists.txt

```diff
@@ -14,11 +14,17 @@ add_library(acdriver SHARED
             Sources/Result.cpp
             Sources/VersionAction.cpp
             Sources/CompileAction.cpp
+            Sources/CompileActionImageSet.cpp
+            Sources/CompileOutput.cpp
```

Comment:
I don't see tests for the catalog object, while there is a ContentsAction that loads it and prints it.

I do have higher level tests that go from json -> car file -> dump_car to check that everything stays consistent. I'll follow up with something in a later pull request that has this coverage with a faked out filesystem.